keanu-python/build.gradle

```diff
@@ -57,9 +57,37 @@ task pytest {
     }
 }
 
+task generatePythonDocumentation {
+    group "pythonDocumentation"
+    doLast {
+        exec {
+            workingDir("../docs/")
+            commandLine "rm", "-rf", "python/"
+        }
+        exec {
+            workingDir("docs/")
+            commandLine "pipenv", "run", "make", "html"
+        }
+        exec {
+            workingDir("docs/")
+            commandLine "pipenv", "run", 'python', 'remove_underscores.py'
```

Comment:
Why do we have to do this?

Reply:
This is because Jekyll won't serve the folders that start with an underscore so in order for the HTML to render properly and to access the right resources, we need to remove the underscores :(. I might add a comment about this though because it's not obvious